backend-api/tests/api-tests/utils/apiInstance.ts

```diff
@@ -9,15 +11,51 @@ function getInstance(baseUrl: string) {
     retryDelay: (retryCount) => retryCount * 200,
   });
   apiInstance.defaults.validateStatus = () => true;
+  axiosRetry(apiInstance, {
+    retries: 2,
+    retryDelay: (retryCount) => retryCount * 200,
+  });
+
+  const interceptor = aws4Interceptor({
+    options: {
+      region: "eu-west-2",
+      service: "execute-api",
+    },
+    credentials: {
+      getCredentials: fromNodeProviderChain({
+        timeout: 1000,
+        maxRetries: 1,
+        profile: process.env.AWS_PROFILE,
+      }),
+    },
+  });
+
+  apiInstance.interceptors.request.use(interceptor);
 
   return apiInstance;
 }
 
 function getSessionsApiInstance() {
-  const sessionsApiUrl = process.env.SESSIONS_API_URL;
-  if (!sessionsApiUrl)
+  const apiUrl = process.env.SESSIONS_API_URL;
```

Comment:
what's the rational for this change ?

Reply:
Why make the `const` name longer? It's inside `getSessionsApiInstance()` so to me it's obvious it's the session API URL. It could even be just `url` to simplify...